### PR TITLE
Memory Arbitration Fuzzer test enable global arbitration

### PR DESCRIPTION
**Pull request by yanngyoung**
Memory Arbitration Fuzzer test enable global arbitration

### CODE REVIEW

File: velox/common/memory/MemoryArbitrator.h [hunk 1]
```diff
@@ -250,6 +250,16 @@ class MemoryArbitrator {
   /// Returns the debug string of this memory arbitrator.
   virtual std::string toString() const = 0;
 
+  /// Set globalArbitrationEnabled to be true
```
Comment (duanmeng) [inline velox/common/memory/MemoryArbitrator.h:253]: Enables global arbitration.

File: velox/common/memory/MemoryArbitrator.h [hunk 2]
```diff
@@ -250,6 +250,16 @@ class MemoryArbitrator {
...
+  void testingEnableGlobalArbitration() {
+    *const_cast<bool*>(&globalArbitrationEnabled_) = true;
+  }
+
+  /// Set globalArbitrationEnabled to be false
```
Comment (duanmeng) [inline velox/common/memory/MemoryArbitrator.h:258]: Disables global arbitration.

File: velox/exec/fuzzer/MemoryArbitrationFuzzer.cpp [hunk 3]
```diff
@@ -751,6 +760,8 @@ void MemoryArbitrationFuzzer::go() {
 
     reSeed();
     ++iteration;
+    // Reset flag
```
Comment (duanmeng) [inline velox/exec/fuzzer/MemoryArbitrationFuzzer.cpp:763]: Revert the flag.

**Review by duanmeng — APPROVED** [review]
LGTM % some nits.

File: velox/common/memory/MemoryArbitrator.h [hunk 4]
```diff
@@ -250,6 +250,16 @@ class MemoryArbitrator {
   /// Returns the debug string of this memory arbitrator.
   virtual std::string toString() const = 0;
 
+  /// Set globalArbitrationEnabled to be true
+  void testingEnableGlobalArbitration() {
```
Comment (xiaoxmeng) [inline velox/common/memory/MemoryArbitrator.h:254]:
```
s/testingEnableGlobalArbitration/testingSetGlobalArbitration(bool)/
```

**Review by xiaoxmeng — APPROVED** [review]
@yanngyoung LGTM % comments from @duanmeng. Thanks!

**Pull request closed without merging**